attestation-agent/kbc/Cargo.toml

```diff
@@ -34,14 +34,15 @@ rstest.workspace = true
 tonic-build = { version = "0.9.2", optional = true }
 
 [features]
-default = ["sample_kbc", "rust-crypto"]
+default = ["sample_kbc", "rust-crypto", "all-attesters", "cc_kbc"]
```

Comment:
Shall still we keep the `cc_kbc` feature optional?

Reply:
yes, I will remove it, this was added for test purpose. :)

Reply:
fixed, pls see [diff](https://github.com/confidential-containers/guest-components/compare/b80f379acbf37ac970727a4ede0fbbc692941b5d..40c4cf43f7cbd23a5b92f421e17b7122fbc7ed19)